src/partials/article.hbs

```diff
@@ -16,3 +16,4 @@ If you typed the URL of this page manually, please double check that you entered
 {{/if}}
 </article>
 {{> article-footer}}
+{{> survey}}
```

Comment:
Adding the survey to the article-footer partial instead of the article partial fixes the banner being too close to the "was this article helpful?" banner, and also removes it from the 404 page.
https://github.com/mulesoft/docs-site-ui/commit/bcb5e865f1d5a2d4386069a0b1aee635d4ad91b9

But does not remove it from the connectors landing page. 
Question: Is the article-footer partial needed in the connectors landing page?

Reply:
Nice find. 

Yes, the article-footer partial is needed to resolve an error described in #99

Reply:
I got a solution to exclude the footers from 404 pages: https://github.com/mulesoft/docs-site-ui/pull/100/commits/b74b86ce899b898a504dd2ba4b8c20502335e4cd

Also, updated CSS to adjust the spacings and I think it looks decent: https://github.com/mulesoft/docs-site-ui/pull/100/commits/cbb75b8ea67890f1d47376cf6b9bc7d8613d39ac